script_tests/test_scripts.py

```diff
@@ -97,6 +97,56 @@ def test_all_in_one_conversion(examples_dir, tmpdir):
     # with --mmap / --load-full-volume
 
 
+def test_sharded_conversion(examples_dir, tmpdir):
+    input_nifti = examples_dir / "Waxholm" / "WHS_SD_rat_T2star_v1.nii.gz"
```

Comment:
Please use a smaller file to avoid bloating the repo (ideally Colin, which is already included in git-lfs on this repo)